common/src/main/java/com/revenuecat/purchases/common/HTTPClient.kt

```diff
@@ -125,7 +128,7 @@ class HTTPClient(
         requestHeaders: Map<String, String>,
         refreshETag: Boolean
     ): HTTPResult? {
-        val jsonBody = body?.convert()
+        val jsonBody = body?.let { mapConverter.convertToJSON(body) }
```

Comment:
just double-checking, android folks: this should be equivalent, right?

Reply:
I would do

```
        val jsonBody = body?.let { mapConverter.convertToJSON(it) }
```

But yes. The difference is that with `it`, you are not using smart casting from kotlin. But it's the same.